BMM.UI.iOS/Application/Implementations/iOSLogger.cs

```diff
@@ -35,19 +43,45 @@ public void Warn(string tag, string message)
         public void Error(string tag, string message)
         {
             WriteToConsole(tag, "Error", message);
-            var dic = new Dictionary<string, string> {{"Tag", tag}, {"Message", message}};
+            var dic = new Dictionary<string, string>
+            {
+                {"Tag", tag},
+                {"Message", message}
+            };
+            AddAnalyticsId(dic);
             Crashes.TrackError(new ErrorWithoutException(tag + " - " + message), dic);
             AppCenterLog.Error(tag, message);
             Microsoft.AppCenter.Analytics.Analytics.TrackEvent("Error without exception", dic);
         }
 
-        public void Error(string tag, string message, Exception exception)
+        public void Error(string tag, string message, Exception exception, bool wasErrorMessagePresentedToUser)
         {
             WriteToConsole(tag, "Error", message + " Exception: " + exception.Message);
-            Crashes.TrackError(exception, new Dictionary<string, string> {{"Tag", tag}, {"Message", message}});
+
+            var parameters = new Dictionary<string, string>
+            {
+                { "Tag", tag },
+                { "Message", message },
+                { "Presented to user", wasErrorMessagePresentedToUser.ToString() }
+            };
+
+            AddAnalyticsId(parameters);
+
+            Crashes.TrackError(exception, parameters); 
             AppCenterLog.Error(tag, message, exception);
-            Microsoft.AppCenter.Analytics.Analytics.TrackEvent("Error with exception",
-                new Dictionary<string, string> {{"Tag", tag}, {"Message", message}, {"Exception", exception.ToString()}, {"StackTrace", exception.StackTrace}});
+            
+            parameters.Add("StackTrace", exception.StackTrace);
```

Comment:
good thinking.
I'm not sure if AppCenter has a character limit on strings, but I would try anyways